.github/workflows/development-workflow.yml

```diff
@@ -15,6 +14,7 @@ env:
     SERVER_IMAGE: prytaneum-server
     NAMESPACE: development
     GRAPHQL_URL: https://dev.prytaneum.io/graphql
+    CI: 1
```

Comment:
Could you add a description of what this is for in a comment?

Reply:
Playwright uses this environment variable to set up some of the configuration differently in the github action. Using it we can get different more detailed output when running the tests locally but it can do a more simplified version when the action runs.

Reply:
That makes sense, please add that to a comment in the file.  Maybe even with a link to the docs where it explains that.